.github/workflows/build-and-test-toolchain.yml

```diff
@@ -82,7 +82,7 @@ jobs:
     runs-on: windows-latest
 
     env:
-      WSLENV: BINUTILS_BRANCH:GCC_BRANCH:MINGW_BRANCH:ARCH:PLATFORM:CRT:TAG:MODULE:FILTER:CCACHE:RUN_BOOTSTRAP:UPDATE_SOURCES:GITHUB_OUTPUT/p
+      WSLENV: BINUTILS_BRANCH:GCC_BRANCH:MINGW_BRANCH:ARCH:PLATFORM:CRT:TAG:MODULE:FILTER:CCACHE:RUN_BOOTSTRAP:UPDATE_SOURCES:GITHUB_OUTPUT:GITHUB_STEP_SUMMARY/p
```

Comment:
The `/p` for `GITHUB_OUTPUT` should stay. E.g. \`WSLENV: GITHUB\_OUTPUT/p:GITHUB\_STEP\_SUMMARY/p\`